hw/drivers/hash/include/hash/sha256_alt.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __SHA256_ALT_H__
+#define __SHA256_ALT_H__
+
+#include <os/mynewt.h>
+
+#if MYNEWT_VAL(MBEDTLS_SHA256_ALT)
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <hash/hash.h>
+
+typedef struct mbedtls_sha256_context
+{
```

Comment:
coding style; open parentheses should be in same line as struct name.

Reply:
done